src/.vscode/launch.json

```diff
@@ -0,0 +1,22 @@
+//debug bash
```

Comment:
you probably want to add this .vscode folder to .gitignore?

Reply:
it could be useful, especially if the launch configurations (like launchsettings in VS) are path-agnostic. I'd like to see the hard-coded C:/ value changed to something relative to the repo (via the `{workspaceRoot}` parameter - then anyone that clones the repo will get a nice f5 experience out of the box!

Reply:
done